sw/host/provisioning/ft_lib/src/lib.rs

```diff
@@ -247,25 +247,63 @@ pub fn run_ft_personalize(
         .into_iter()
         .take(certs.cdi_1_certificate_size)
         .collect();
+    let tpm_ek_tbs_cert_bytes: Vec<u8> = certs
+        .tpm_ek_tbs_certificate
+        .clone()
+        .into_iter()
+        .take(certs.tpm_ek_tbs_certificate_size)
+        .collect();
+    let tpm_cek_tbs_cert_bytes: Vec<u8> = certs
+        .tpm_cek_tbs_certificate
+        .clone()
+        .into_iter()
+        .take(certs.tpm_cek_tbs_certificate_size)
+        .collect();
+    let tpm_cik_tbs_cert_bytes: Vec<u8> = certs
+        .tpm_cik_tbs_certificate
+        .clone()
+        .into_iter()
+        .take(certs.tpm_cik_tbs_certificate_size)
+        .collect();
 
     // Log certificates to console and check they are parsable with OpenSSL.
     let cert_endorsement_sk = SecretKey::<NistP256>::read_pkcs8_der_file(cert_endorsement_ecc_sk)?;
     let uds_cert_bytes = parse_and_endorse_uds_cert(uds_tbs_cert_bytes, &cert_endorsement_sk)?;
+    let tpm_ek_cert_bytes =
+        parse_and_endorse_uds_cert(tpm_ek_tbs_cert_bytes, &cert_endorsement_sk)?;
```

Comment:
maybe `parse_and_endorse_uds_cert` should be rename now that it is used for other certificates than the UDS?